nexus/src/db/datastore/project.rs

```diff
@@ -62,7 +63,17 @@ impl DataStore {
                     ErrorHandler::Conflict(ResourceType::Project, &name),
                 )
             }
-        })
+        })?;
+
+        // Create resource usage for the project.
+        //
+        // NOTE: if you do this before the project is created, it'll exist as
```

Comment:
Same as for Organizations.

Reply:
See: 83d8d9fc97ff8c8991e0e7f1fde898efe5e54506